client/src/components/Comment/Comment.css

```diff
@@ -0,0 +1,67 @@
+.Comment_body {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 50rem;
+  height: auto;
+  background-color: var(--gray-dark);
+  font-size: 14px;
+  font-family: var(--paragraph-font);
+  padding: 1.2rem;
+}
+
+.Comment_h4 {
+  margin: 0;
+}
+
+.Comment_section {
+  margin: 0.7rem;
+  height: 90%;
+  margin-bottom: 0;
+}
+
+.Comment_section_1 {
```

Comment:
Same here - a semantic name. Also, you never want to put numbers in a class name. This is where you'd use modifiers on a block(re BEM in the readme)

Reply:
Also, can you make these classnames in line with the naming convention?

Reply:
Two underscores, two dashes, etc